arangod/Aql/ExecutionEngine.cpp

```diff
@@ -593,19 +593,23 @@ struct DistributedQueryInstanciator final
 
       for (auto const& [server, queryId, rebootId] : srvrQryId) {
         TRI_ASSERT(!server.starts_with("server:"));
-        std::function<void(void)> f = [srvr = server, id = _query.id(),
-                                       vn = _query.vocbase().name(), &df]() {
-          LOG_TOPIC("d2554", INFO, Logger::QUERIES)
-              << "killing query " << id << " because participating DB server "
-              << srvr << " is unavailable";
-          try {
-            methods::Queries::kill(df, vn, id);
-          } catch (...) {
-            // it does not really matter if this fails.
-            // if the coordinator contacts the failed DB server next time, it
-            // will realize it has failed.
-          }
-        };
+        std::function<void(void)> f =
+            [srvr = server, id = _query.id(), vn = _query.vocbase().name(), &df,
+             qs = _query.queryString(),
```

Comment:
This change is independent of the rest of the PR, it adds the query string and parameters in case of a failed query to the logs.